doc/conf.py

```diff
@@ -207,8 +207,8 @@
 html_title = project
 html_short_title = project
 html_baseurl = f"{doc_url}/dev/" if isdev else f"{doc_url}/latest/"
-html_logo = ""
-html_favicon = "_static/favicon.png"
+html_logo = "https://github.com/sfrooti/PyGMT-Logo/blob/75add2ab7fc522bcfd5ce199e7cc73e82bbab910/PyGMT_Transparent_Horizontal.png"
+html_favicon = "https://github.com/sfrooti/PyGMT-Logo/blob/75add2ab7fc522bcfd5ce199e7cc73e82bbab910/PyGMT_Transparent_Horizontal.png"
```

Comment:
Did not test it, but maybe it is similar to the External Resources page where it has to be ``raw`` instead of ``blob`` in the URL?
https://github.com/GenericMappingTools/pygmt/blob/e7abf7ce5c806815d6d922092814b98ebd173878/doc/external_resources.md?plain=1#L105-L111
or
https://github.com/GenericMappingTools/pygmt/blob/e7abf7ce5c806815d6d922092814b98ebd173878/doc/external_resources.md?plain=1#L150-L157

Why is there this numer-letter combination "75add2ab7fc522bcfd5ce199e7cc73e82bbab910" (maybe refering to a commit) and not just the "main" branch used in the URL? 

```suggestion
html_logo = "https://github.com/sfrooti/PyGMT-Logo/raw/main/PyGMT_Transparent_Horizontal.png"
html_favicon = "https://github.com/sfrooti/PyGMT-Logo/raw/main/PyGMT_Transparent_Horizontal.png"
```